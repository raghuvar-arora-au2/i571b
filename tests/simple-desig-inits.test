{ [2] = 99, [5] = 44 }
