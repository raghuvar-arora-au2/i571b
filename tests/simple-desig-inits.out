[
   0,
   0,
   99,
   0,
   0,
   44
]
